components/footer/footer.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import useTheme from '../use-theme'
+import FooterGroup from './footer-group'
+import FooterLink from './footer-link'
+import FooterColumn from './footer-column'
+import { FooterContext } from './footer-context'
+
+interface Props {
+  light?: boolean
+  subFooter?: string | React.ReactNode
+  ariaLabel?: string
+  maxWidth?: string
+  breakPoint?: string
+}
+
+const defaultProps = {
+  light: false,
+  ariaLabel: 'Footer Links',
+  breakPoint: '960px',
+  maxWidth: '1000px',
+}
+
+type NativeAttrs = Omit<React.HTMLAttributes<any>, keyof Props>
+export type FooterProps = Props & typeof defaultProps & NativeAttrs
+
+const Footer: React.FC<React.PropsWithChildren<FooterProps>> = ({
+  light,
+  ariaLabel,
+  children,
+  subFooter,
+  maxWidth,
+  breakPoint,
+  ...props
+}) => {
```

Comment:
For Footer, I guess there are also users who might want to hover at the bottom of the page.